cloudgenix_config/__init__.py

```diff
@@ -403,6 +404,11 @@ def build_lookup_dict(list_content, key_val='name', value_val='id', force_nag=Fa
         item_value = item.get(value_val)
         # print(item_key, item_value)
         if item_key and item_value is not None:
+            # Below check to handle lookup in ion 9k
+            # 9k can have same port and bypasspair names
+            # Adding '_bypasspair' for bypasspairs in range 12-15
+            if key_val == 'name' and model_name == 'ion 9000' and item.get('type') == 'bypasspair' and int(item_key) in range(12,17):
```

Comment:
Special look up dict built for bypasspair ports in range(12,17) only. Some conversions further down in the code don't have a check for this range and the key will not exist, causing an error in the lookup.

Reply:
As discussed removed the range check and adding parent_type for all bypasspairs